### ISSUE TITLE
ECS Fields are not mapped properly in package mappings

### ISSUE THREAD

**Issue by joeperuzzi** [elastic/elastic-package#1209]
We have defined in ecs.yml in our fields directory, but they are not reflecting when we install the plugin into Kibana:

- external: ecs
  name: vulnerability.id
- external: ecs
  name: event.id
- external: ecs
  name: vulnerability.result
- external: ecs
  name: vulnerability.reference
- external: ecs
  name: vulnerability.score.base
- external: ecs
  name: vulnerability.score.version

Highlighting "vulnerability.score.base", which should be a float under ecs, is being mapped from our plugin as a keyword field instead:  

<img width="227" alt="image" src="https://user-images.githubusercontent.com/122561341/230198841-7535a646-ea66-40b7-b7c4-87b365625d0b.png">

<img width="314" alt="image" src="https://user-images.githubusercontent.com/122561341/230199144-264b924d-ca3f-43db-b496-6b9a53873f1f.png">

I'm not sure if this is a bug or just a misconfiguration but any help you could give us would be great!

Just for quick reference from the ticket:
https://www.elastic.co/guide/en/ecs/current/ecs-vulnerability.html

**Comment by jsoriano** [comment]
Hey @joeperuzzi,

What version of elastic-package are you using?
And what version of ECS? (as defined in `_dev/build/build.yml`)

**Comment by joeperuzzi** [comment]
@jsoriano we are unaware of where _dev/build/build.yml may be located.  I'm checking on the version, but its not the latest.

**Comment by jsoriano** [comment]
Yes, sorry, I should have pointed to https://github.com/elastic/elastic-package/blob/main/docs/howto/dependency_management.md directly :slightly_smiling_face:

You can find some examples of these files in the integrations repository, such as this one for apache: https://github.com/elastic/integrations/blob/main/packages/apache/_dev/build/build.yml
